src/main/java/com/ipiecoles/java/java350/model/Employe.java

```diff
@@ -68,24 +68,43 @@ public Integer getNbRtt(){
     }
 
     public Integer getNbRtt(LocalDate d){
-        int i1 = d.isLeapYear() ? 365 : 366;
-        int var = 104;
+        int nbJoursAnnee = d.isLeapYear() ? 366 : 365;
+        int nbJourSameDimancheAnnee = 104;
         switch (LocalDate.of(d.getYear(),1,1).getDayOfWeek()){
-            case THURSDAY: if(d.isLeapYear()) var =  var + 1; break;
-            case FRIDAY: if(d.isLeapYear()) var =  var + 2; else var =  var + 1;
-            case SATURDAY: var = var + 1; break;
+            case THURSDAY:
+                if(d.isLeapYear())
+                {
+                    nbJourSameDimancheAnnee =  nbJourSameDimancheAnnee + 1;
+                    break;
+                }
+            case FRIDAY:
+                if(d.isLeapYear())
+                {
+                    nbJourSameDimancheAnnee =  nbJourSameDimancheAnnee + 2;
+                    break;
+                } else{ nbJourSameDimancheAnnee =  nbJourSameDimancheAnnee + 1;}
```

Comment:
Utilise la même indentation partour
```suggestion
                } else { 
                    nbJourSameDimancheAnnee =  nbJourSameDimancheAnnee + 1;
                }
```